apps/api/prisma/schema.prisma

```diff
@@ -292,14 +292,27 @@ model ZoomLevel {
 }
 
 model Folder {
-  id             Int      @id @default(autoincrement())
+  id             Int        @id @default(autoincrement())
   displayNameEn  String
   displayOrder   Int?
   parentFolderId Int?
-  parentFolder   Folder?  @relation("FolderTree", fields: [parentFolderId], references: [id], onDelete: NoAction, onUpdate: NoAction)
-  childFolders   Folder[] @relation("FolderTree")
-  case           Case?    @relation(fields: [caseId], references: [id])
+  parentFolder   Folder?    @relation("FolderTree", fields: [parentFolderId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  childFolders   Folder[]   @relation("FolderTree")
+  case           Case?      @relation(fields: [caseId], references: [id])
   caseId         Int?
+  document       Document[]
 
   @@unique([caseId, displayNameEn, parentFolderId])
 }
+
+model Document {
+  guid                 String  @id @default(uuid())
+  name                 String
+  folder               Folder  @relation(fields: [folderId], references: [id])
+  folderId             Int
+  blobStorageContainer String?
+  blobStoragePath      String?
+  status               String  @default("waiting_for_upload")
+
+  @@unique([name, folderId])
+}
```

Comment:
branch 518 does not have these changes .. for me at least, so I had to add them

Reply:
The way to tackle this, would be to rebase `518` with `main`, and then rebase `537` with `518`. If we commit the same work twice, especially if we are talking about two database migrations, then our migrations are going to fail. For example here, we are going to have two migrations that are going to try and create the `Document` table. This will fail, because we can't have two tables with the same name.